sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -1124,6 +1129,29 @@ public enum JsonPropertyNames
         UseExact = 0,
         CamelCase = 1,
     }
+    public partial class ModelJsonConverter : System.Text.Json.Serialization.JsonConverter<Azure.Core.Serialization.IModelSerializable>
+    {
+        public ModelJsonConverter() { }
+        public ModelJsonConverter(bool ignoreAdditionalProperties) { }
+        public bool IgnoreAdditionalProperties { get { throw null; } set { } }
```

Comment:
Would it make sense to align naming with JsonSerializerOptions?  e.g. call this "IgnoreUnmappedMembers"?  See [JsonSerializerOptions.IgnoreUnmappedMemgers](https://learn.microsoft.com/en-us/dotnet/api/system.text.json.jsonserializeroptions.unmappedmemberhandling?view=net-8.0#system-text-json-jsonserializeroptions-unmappedmemberhandling)

Reply:
this was changed to just a `Format` property of "D" or "W" for Data or Wire format.  Data format will not ignore readonly properties, but Wire format will.